zstacklib/zstacklib/utils/iproute.py

```
@@ -671,14 +735,16 @@ def add_namespace(namespace):
     try:
         pyroute2.netns.create(namespace)
     except OSError as e:
-        if e.errno == os.errno.EEXIST: # namespace already exists
+        if e.errno == os.errno.EEXIST:  # namespace already exists
```

Pylint (3.3.7)

[error] Module 'os' has no 'errno' member

(E1101)